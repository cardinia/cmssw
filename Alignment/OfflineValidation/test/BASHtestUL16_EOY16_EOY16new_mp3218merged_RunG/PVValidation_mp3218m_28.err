26-Sep-2019 13:07:50 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016G/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/00000/0C321742-6612-E911-8D0A-003048FFD75A.root
26-Sep-2019 13:07:54 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016G/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/00000/0C321742-6612-E911-8D0A-003048FFD75A.root
Begin processing the 1st record. Run 279658, Event 814126612, LumiSection 455 on stream 0 at 26-Sep-2019 13:08:01.946 CEST
Begin processing the 1001st record. Run 279658, Event 822575297, LumiSection 459 on stream 0 at 26-Sep-2019 13:11:44.121 CEST
Begin processing the 2001st record. Run 279658, Event 829812135, LumiSection 463 on stream 0 at 26-Sep-2019 13:15:04.699 CEST
Begin processing the 3001st record. Run 279658, Event 837776376, LumiSection 467 on stream 0 at 26-Sep-2019 13:18:21.458 CEST
Begin processing the 4001st record. Run 279658, Event 844184831, LumiSection 471 on stream 0 at 26-Sep-2019 13:21:33.378 CEST
Begin processing the 5001st record. Run 279658, Event 852589826, LumiSection 475 on stream 0 at 26-Sep-2019 13:24:42.439 CEST
Begin processing the 6001st record. Run 279658, Event 859516369, LumiSection 479 on stream 0 at 26-Sep-2019 13:27:58.061 CEST
Begin processing the 7001st record. Run 279658, Event 867854355, LumiSection 483 on stream 0 at 26-Sep-2019 13:31:13.709 CEST
Begin processing the 8001st record. Run 279658, Event 873806979, LumiSection 487 on stream 0 at 26-Sep-2019 13:34:32.486 CEST
Begin processing the 9001st record. Run 279658, Event 883173485, LumiSection 492 on stream 0 at 26-Sep-2019 13:38:06.170 CEST
Begin processing the 10001st record. Run 279658, Event 891207583, LumiSection 496 on stream 0 at 26-Sep-2019 13:41:25.484 CEST
Begin processing the 11001st record. Run 279658, Event 899382306, LumiSection 500 on stream 0 at 26-Sep-2019 13:44:23.143 CEST
Begin processing the 12001st record. Run 279658, Event 906466564, LumiSection 504 on stream 0 at 26-Sep-2019 13:47:28.848 CEST
Begin processing the 13001st record. Run 279658, Event 915749001, LumiSection 509 on stream 0 at 26-Sep-2019 13:50:26.219 CEST
Begin processing the 14001st record. Run 279658, Event 923752175, LumiSection 513 on stream 0 at 26-Sep-2019 13:53:33.598 CEST
Begin processing the 15001st record. Run 279658, Event 931342330, LumiSection 517 on stream 0 at 26-Sep-2019 13:56:33.709 CEST
Begin processing the 16001st record. Run 279658, Event 938841746, LumiSection 521 on stream 0 at 26-Sep-2019 13:59:43.297 CEST
Begin processing the 17001st record. Run 279658, Event 945478840, LumiSection 525 on stream 0 at 26-Sep-2019 14:02:42.692 CEST
Begin processing the 18001st record. Run 279658, Event 955407412, LumiSection 530 on stream 0 at 26-Sep-2019 14:05:38.899 CEST
Begin processing the 19001st record. Run 279658, Event 963445889, LumiSection 534 on stream 0 at 26-Sep-2019 14:08:35.115 CEST
Begin processing the 20001st record. Run 279658, Event 970275083, LumiSection 538 on stream 0 at 26-Sep-2019 14:11:37.345 CEST
Begin processing the 21001st record. Run 279658, Event 976804760, LumiSection 542 on stream 0 at 26-Sep-2019 14:14:40.507 CEST
Begin processing the 22001st record. Run 279658, Event 985217398, LumiSection 546 on stream 0 at 26-Sep-2019 14:17:43.283 CEST
Begin processing the 23001st record. Run 279658, Event 994661111, LumiSection 551 on stream 0 at 26-Sep-2019 14:20:44.645 CEST
Begin processing the 24001st record. Run 279658, Event 1001318266, LumiSection 555 on stream 0 at 26-Sep-2019 14:23:46.229 CEST
Begin processing the 25001st record. Run 279658, Event 1008469989, LumiSection 559 on stream 0 at 26-Sep-2019 14:26:25.177 CEST
Begin processing the 26001st record. Run 279658, Event 1016281687, LumiSection 563 on stream 0 at 26-Sep-2019 14:29:23.684 CEST
Begin processing the 27001st record. Run 279658, Event 1025336432, LumiSection 568 on stream 0 at 26-Sep-2019 14:32:27.521 CEST
Begin processing the 28001st record. Run 279658, Event 1031224831, LumiSection 572 on stream 0 at 26-Sep-2019 14:35:38.849 CEST
Begin processing the 29001st record. Run 279658, Event 1039714987, LumiSection 576 on stream 0 at 26-Sep-2019 14:38:38.247 CEST
Begin processing the 30001st record. Run 279658, Event 1048851085, LumiSection 581 on stream 0 at 26-Sep-2019 14:42:02.399 CEST
Begin processing the 31001st record. Run 279658, Event 1055926452, LumiSection 585 on stream 0 at 26-Sep-2019 14:45:22.102 CEST
Begin processing the 32001st record. Run 279658, Event 1063609159, LumiSection 589 on stream 0 at 26-Sep-2019 14:48:27.727 CEST
Begin processing the 33001st record. Run 279658, Event 1072171375, LumiSection 594 on stream 0 at 26-Sep-2019 14:51:30.509 CEST
Begin processing the 34001st record. Run 279658, Event 1078649223, LumiSection 598 on stream 0 at 26-Sep-2019 14:54:15.544 CEST
Begin processing the 35001st record. Run 279993, Event 43023786, LumiSection 88 on stream 0 at 26-Sep-2019 14:56:04.664 CEST
Begin processing the 36001st record. Run 279993, Event 51009023, LumiSection 92 on stream 0 at 26-Sep-2019 14:56:08.557 CEST
Begin processing the 37001st record. Run 279993, Event 58256594, LumiSection 96 on stream 0 at 26-Sep-2019 14:56:11.478 CEST
Begin processing the 38001st record. Run 279993, Event 67115279, LumiSection 100 on stream 0 at 26-Sep-2019 14:56:14.264 CEST
Begin processing the 39001st record. Run 279993, Event 74032046, LumiSection 104 on stream 0 at 26-Sep-2019 14:56:17.903 CEST
Begin processing the 40001st record. Run 279993, Event 82343789, LumiSection 108 on stream 0 at 26-Sep-2019 14:56:20.843 CEST
Begin processing the 41001st record. Run 279993, Event 90452348, LumiSection 112 on stream 0 at 26-Sep-2019 14:56:23.490 CEST
Begin processing the 42001st record. Run 279993, Event 100056671, LumiSection 117 on stream 0 at 26-Sep-2019 14:56:26.335 CEST
Begin processing the 43001st record. Run 279993, Event 107524481, LumiSection 121 on stream 0 at 26-Sep-2019 14:56:29.002 CEST
Begin processing the 44001st record. Run 279993, Event 114364344, LumiSection 125 on stream 0 at 26-Sep-2019 14:56:31.760 CEST
Begin processing the 45001st record. Run 279993, Event 122035709, LumiSection 129 on stream 0 at 26-Sep-2019 14:56:34.335 CEST
Begin processing the 46001st record. Run 279993, Event 130869926, LumiSection 133 on stream 0 at 26-Sep-2019 14:56:37.100 CEST
Begin processing the 47001st record. Run 279993, Event 139109608, LumiSection 138 on stream 0 at 26-Sep-2019 14:56:40.764 CEST
Begin processing the 48001st record. Run 279993, Event 147603194, LumiSection 143 on stream 0 at 26-Sep-2019 14:56:43.878 CEST
Begin processing the 49001st record. Run 279993, Event 155536656, LumiSection 151 on stream 0 at 26-Sep-2019 14:56:46.519 CEST
26-Sep-2019 14:56:48 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016G/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/00000/0C321742-6612-E911-8D0A-003048FFD75A.root
26-Sep-2019 14:56:48 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016G/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/00000/76EBB6EA-1D12-E911-AC1A-0CC47A7C34EE.root
26-Sep-2019 14:56:50 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016G/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/00000/76EBB6EA-1D12-E911-AC1A-0CC47A7C34EE.root
26-Sep-2019 14:57:42 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016G/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/00000/76EBB6EA-1D12-E911-AC1A-0CC47A7C34EE.root

=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 fileAction           -s file_close                             2        2
    2 fileAction           -s file_open                              4        4

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 fileAction           PostProcessEvent PostGlobalEndRun 
    2 fileAction           pre-events       pre-events       PostProcessEvent

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
System                  6                   6

dropped waiting message count 0
[9.315MB/9.315MB][100%][==================================================][9.315MB/s]  [9.315MB/9.315MB][100%][==================================================][9.315MB/s]  
