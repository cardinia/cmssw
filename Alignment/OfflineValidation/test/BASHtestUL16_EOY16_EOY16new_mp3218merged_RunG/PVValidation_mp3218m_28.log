000 (3189587.028.000) 09/26 11:25:05 Job submitted from host: <137.138.44.75:9618?addrs=137.138.44.75-9618+[2001-1458-d00-1a--b]-9618&alias=bigbird14.cern.ch&noUDP&sock=schedd_2873_37ae_40>
...
040 (3189587.028.000) 09/26 13:07:37 Started transferring input files
	Transferring to host: <188.184.139.148:9618?addrs=188.184.139.148-9618+[2001-1458-301-45--100-199]-9618&noUDP&sock=31934_244c_128871>
...
040 (3189587.028.000) 09/26 13:07:37 Finished transferring input files
...
001 (3189587.028.000) 09/26 13:07:37 Job executing on host: <188.184.139.148:9618?addrs=188.184.139.148-9618+[2001-1458-301-45--100-199]-9618&noUDP&sock=5616_aff6_3>
...
006 (3189587.028.000) 09/26 13:07:46 Image size of job updated: 154280
	92  -  MemoryUsage of job (MB)
	93236  -  ResidentSetSize of job (KB)
...
006 (3189587.028.000) 09/26 13:12:47 Image size of job updated: 654372
	535  -  MemoryUsage of job (MB)
	547468  -  ResidentSetSize of job (KB)
...
006 (3189587.028.000) 09/26 13:17:48 Image size of job updated: 656144
	537  -  MemoryUsage of job (MB)
	549240  -  ResidentSetSize of job (KB)
...
006 (3189587.028.000) 09/26 13:27:49 Image size of job updated: 656616
	537  -  MemoryUsage of job (MB)
	549712  -  ResidentSetSize of job (KB)
...
006 (3189587.028.000) 09/26 13:32:50 Image size of job updated: 656768
	537  -  MemoryUsage of job (MB)
	549848  -  ResidentSetSize of job (KB)
...
006 (3189587.028.000) 09/26 14:17:53 Image size of job updated: 657664
	538  -  MemoryUsage of job (MB)
	550744  -  ResidentSetSize of job (KB)
...
040 (3189587.028.000) 09/26 14:58:02 Started transferring output files
...
040 (3189587.028.000) 09/26 14:58:02 Finished transferring output files
...
005 (3189587.028.000) 09/26 14:58:02 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 01:47:40, Sys 0 00:00:14  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 01:47:40, Sys 0 00:00:14  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	9603  -  Run Bytes Sent By Job
	1015  -  Run Bytes Received By Job
	9603  -  Total Bytes Sent By Job
	1015  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1           
	   Disk (KB)            :     9587        1   2894108 
	   Memory (MB)          :      538     2000      2000 
...
