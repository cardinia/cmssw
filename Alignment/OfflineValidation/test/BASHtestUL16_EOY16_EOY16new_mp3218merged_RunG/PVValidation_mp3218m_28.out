Job started at  Thu Sep 26 13:07:37 CEST 2019
batch dir: /pool/condor/dir_18276 release: /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/alignmentObjects/paconnor/CMSSW_10_6_0/src release base: /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_0
cmsRun PVValidation_mp3218m_279658_cfg.py
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is DATA!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Applying TrackerSurfaceDeformations!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Running DA Algorithm!
Content of working dir is total 9.4M -rw-r--r--. 1 tkello zh 9.4M Sep 26 14:58 PVValidation_mp3218m_279658.root -rw-r--r--. 1 tkello zh 21K Sep 26 13:07 PVValidation_mp3218m_279658_cfg.py -rw-r--r--. 1 tkello zh 8.2K Sep 26 14:58 _condor_stderr -rw-r--r--. 1 tkello zh 522 Sep 26 13:07 _condor_stdout -rwxr-xr-x. 1 tkello zh 1015 Sep 26 13:07 condor_exec.exe -rw-------. 1 tkello zh 4.3K Sep 26 13:07 tkello.cc drwx------. 3 tkello zh 4.0K Sep 26 13:07 tmp drwx------. 3 tkello zh 4.0K Sep 26 13:07 var
Job ended at  Thu Sep 26 14:58:02 CEST 2019
